.eslintrc

```diff
@@ -158,6 +158,18 @@
     }
   },
   "overrides": [
+    {
+      "files": ["*.jsx", "*.tsx"],
+      "rules": {
+        "no-unconditional-metabase-links-render": "error"
+      }
+    },
+    {
+      "files": ["*.unit.spec.jsx", "*.unit.spec.tsx"],
```

Comment:
Some unit tests will contain the assertions for Metabase learn and docs links, so we need to disable it.